app/controllers/users_controller.rb

```diff
@@ -1,9 +1,10 @@
 class UsersController < ApplicationController
   def index
-    @user = User.all
+    @author = User.all
   end
 
   def show
     @user_id = params[:id]
+    @user = User.where(id: @user_id)
```

Comment:
- [ ] Let's use the `find_by` method instead so that it returns one object instead an array of one object. This way in the user show view we won't need be using `@user[0]`.

  ```
  @user = User.find_by(id: params[:id])
  ```